Libraries/Opc.Ua.Client/Session.cs

```diff
@@ -767,6 +791,28 @@ public int GoodPublishRequestCount
                 }
             }
         }
+
+        /// <summary>
+        /// Gets and sets the minimum number of publish requests to be used in the session.
+        /// </summary>
+        public int MinPublishRequestCount
+        {
+            get => m_minPublishRequestCount;
+            set
+            {
+                lock (SyncRoot)
+                {
+                    if (value >= 1 && value <= kMaxPublishRequestCount)
```

Comment:
unreleated: how do you came up with 100 for kMaxPublishRequestCount?

Reply:
btw: would kMinPublishRequestCountMax be a better name?

Reply:
indeed